src/component/Account/Account.js

```diff
@@ -6,34 +6,42 @@ import { ContractContext } from "../../hooks/contract";
 
 import Spinner from "../utils/Spinner";
 import AccountCard from "./AccountCard/AccountCard";
+import Generation from "../Generation/Generation"
 
 export default () => {
   const nearContext = useContext(NearContext);
   const useContract = useContext(ContractContext);
-  const { corgis, loading } = useContract;
-  useEffect(() => window.localStorage.removeItem("create"));
+  const { loading, getCorgisList } = useContract;
+  let corgiUpdateKey = 0;
+  let {corgis} = useContract;
+  let Corgis;
+  
+   useEffect(() => {
+       corgis = getCorgisList(nearContext.user.accountId);
```

Comment:
this should trigger an error, you already have corgis this line:   `let {corgis} = useContract;` after you define corgis here. 
this is the warning: Assignments to the 'corgis' variable from inside React Hook useEffect will be lost after each render. To preserve the value over time, store it in a useRef Hook and keep the mutable value in the '.current' property. Otherwise, you can move this variable directly inside useEffect.